infrastructure/cli/environment.go

```diff
@@ -64,8 +65,11 @@ func AppendCliEnvironmentVariables(currentEnv []string, appendToken bool) (updat
 	if appendToken {
 		// there can only be one - highlander principle
 		if currentConfig.AuthenticationMethod() == lsp.OAuthAuthentication {
-			updatedEnv = append(updatedEnv, auth.CONFIG_KEY_OAUTH_TOKEN+"="+currentConfig.Token())
-			updatedEnv = append(updatedEnv, strings.ToUpper(configuration.FF_OAUTH_AUTH_FLOW_ENABLED+"=1"))
+			oAuthToken := currentConfig.TokenAsOAuthToken()
+			if len(oAuthToken.AccessToken) > 0 {
+				updatedEnv = append(updatedEnv, configuration.AUTHENTICATION_BEARER_TOKEN+"="+oAuthToken.AccessToken)
+				updatedEnv = append(updatedEnv, strings.ToUpper(configuration.FF_OAUTH_AUTH_FLOW_ENABLED+"=1"))
```

Comment:
note: the feature flag is not required if the oauth access token is given via `SNYK_OAUTH_TOKEN`